packages/fast-usdc/src/pool-share-math.js

```diff
@@ -120,3 +125,66 @@ export const withFees = (shareWorth, fees) => {
   const balancePost = add(shareWorth.numerator, fees);
   return makeRatioFromAmounts(balancePost, shareWorth.denominator);
 };
+
+/**
+ *
+ * @param {Amount<'nat'>} requested
+ * @param {Amount<'nat'>} poolSeatAllocation
+ * @param {Amount<'nat'>} encumberedBalance
+ * @param {PoolStats} poolStats
+ * @throws {Error} if conditions are not met
```

Comment:
Let's either say what conditions or skip this `@throws`.

```suggestion
 * @throws {Error} if too much is requested
```